src/sandboxes/base.py

```
@@ -0,0 +1,301 @@
+            try:
+                return json.loads(stdout.decode("utf-8"))
+            except json.JSONDecodeError:
+                return {
+                    "success": False,
+                    "error": "Failed to parse worker output",
+                    "stdout": stdout.decode("utf-8", errors="replace"),
+                }
+
+        except asyncio.CancelledError:
+            raise
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)